packages/kitchen-sink/src/style.css

```diff
@@ -901,6 +901,24 @@ input[type='range'] {
   background-color: #45a049;
 }
 
+.markdown-editor-container {
+  width: 48%;
```

Comment:
https://github.com/aidenybai/million/assets/96334363/1741dc30-6823-4cc4-b5fd-41374d2858e4

There is still this issue there @UzairNoman. Can you please review it?